.github/actions/test/action.yaml

```diff
@@ -7,5 +7,13 @@ runs:
     run: cargo test --manifest-path tools/slicec-cs/Cargo.toml
     shell: bash
   - name: 🧪 Test
-    run: dotnet test --no-build --verbosity normal --blame-hang-timeout 8m
+    # See https://github.com/microsoft/vstest/issues/2080#issuecomment-539879345
+    run: dotnet test --no-build --verbosity normal --blame-hang-timeout 10s --logger:"console;noprogress=true"
```

Comment:
How long does a single test usually take? 10s feels on the short side to me, but to be fair, I don't really have a scale for how quickly we expect these tests to complete...

Reply:
It's for a single unit test. So it should be much faster than 10s